CLAUDE.md

```
@@ -0,0 +1,21 @@
+- `bun run test` — Run frontend tests
+
+## Stack
+
+- Rust backend, TypeScript frontend
+- Typst for document rendering
+- Docker for containerized builds
+
+## Standards
+
+- Use `lintro` for linting (`lintro chk`, `lintro fmt`)
+- Use `bun` instead of `npm`
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)